census_view.cpp

```diff
@@ -1589,7 +2580,28 @@ void CensusView::UponAddCell(wxCommandEvent&)
     status() << "Add: " << timer.stop().elapsed_msec_str() << std::flush;
 }
 
-void CensusView::UponDeleteCells(wxCommandEvent&)
+void CensusGridView::UponAddCell(wxCommandEvent&)
+{
+    wxBusyCursor reverie;
+    Timer timer;
+
+    cell_parms().push_back(case_parms()[0]);
+
+    grid_window_->ClearSelection();
```

Comment:
Why do we need to do this? Looking at #121, I see that this wasn't previously done.

Reply:
DVC automatically selects just added row so I remove `ClearSelection` and add the code that selects the added row for the grid.